internal/cmd/kafka/command_topic.go

```diff
@@ -134,15 +134,15 @@ func (c *authenticatedTopicCommand) getNumPartitions(topicName string) (int, err
 		return 0, err
 	}
 
-	partitionsResp, httpResp, err := kafkaREST.CloudClient.ListKafkaPartitions(kafkaClusterConfig.ID, topicName)
+	topicResp, httpResp, err := kafkaREST.CloudClient.GetKafkaTopic(kafkaClusterConfig.ID, topicName)
 	if err != nil {
 		if restErr, parseErr := kafkarest.ParseOpenAPIErrorCloud(err); parseErr == nil && restErr.Code == ccloudv2.UnknownTopicOrPartitionErrorCode {
 			return 0, fmt.Errorf(errors.UnknownTopicErrorMsg, topicName)
 		}
 		return 0, kafkarest.NewError(kafkaREST.CloudClient.GetUrl(), err, httpResp)
 	}
 
-	return len(partitionsResp.Data), nil
+	return int(topicResp.PartitionsCount), nil
```

Comment:
Do we need to convert this to an `int`? It looks like all of the APIs using it want an `int32`.